libs/custom-scanner/src/custom_a4_scanner.ts

```diff
@@ -247,6 +247,13 @@ export class CustomA4Scanner implements CustomScanner {
         this.withRetries(async () => {
           (await this.createJobInternal()).okOrElse(fail);
 
+          // To stop the motors, the createJobInternal call above is sufficient, as it ends the
+          // current job, if any, before creating a new one. This in and of itself stops the
+          // motors.
```

Comment:
I spent a good bit of time with the scanner's "manuale comandi" before landing on this approach haha. I also looked into the CANC (cancel) and STOP (stop scan) commands, and while those will stop the motors during a scan operation, they expectedly don't stop the motors outside of that context. JOB end appears to be the best way to do so. And also nicely requires the fewest code modifications

Reply:
Nice find. Does creating this "empty" job have any side effects afterwards? Does the scanner expect us to send some movement command as part of the job?

Reply:
Good q! As far as I can tell, it doesn't. Here's what the command manual has to say about the JOB command:

<table>
<img width="860" alt="job-command" src="https://github.com/votingworks/vxsuite/assets/12616928/96d29f03-e29b-49ff-b626-f78c8f300296">
</table>

The other question on my mind has been whether it's okay to leave jobs open like we do. But we've already been doing that, so this PR doesn't change anything on that front.

I've also been wondering whether it would make more sense for us to open a job once and keep using it instead of constantly opening and closing jobs. But what we've been doing has been working, and I don't want to make any risky changes.

Reply:
I think the concept of "jobs" in the Custom scanner is essentially meaningless. It always returns the same job ID and is pretty loose about job handling. It does want to have _some_ handling of jobs, though. I tried to build it in a sane way initially, but it didn't work so I just copied what their driver code did.